backend/clubs/views.py

```diff
@@ -2878,33 +2878,64 @@ def get(self, request):
                                     type: integer
         ---
         """
-        id = request.query_params.get("event")
-        if not id:
-            return Response({"count": 0})
-
-        try:
-            conn = get_redis_connection()
-        except NotImplementedError:
-            return Response({"count": 0})
-        key = f"zoom:meeting:live:{id}"
-        ans = conn.get(key)
-        if ans is None:
-            ans = 0
-        else:
-            ans = int(ans)
-        return Response({"count": ans})
+        meeting_id = request.query_params.get("event")
+        return Response({"count": ZoomMeetingVisit.objects.filter(meeting_id=meeting_id).count()})
```

Comment:
Right now this is the same functionality as before, but can be pretty easily expanded upon so that we can list users within the meeting if that's something we decide will be helpful to prospective applicants